api4/team_test.go

```diff
@@ -582,6 +584,11 @@ func TestUpdateTeamPrivacy(t *testing.T) {
 			}
 			require.Equal(t, test.wantType, team.Type)
 			require.Equal(t, test.wantOpenInvite, team.AllowOpenInvite)
+			if test.wantInviteIdChanged {
+				require.NotEqual(t, test.originalInviteId, team.InviteId)
+			} else {
+				require.Equal(t, test.originalInviteId, team.InviteId)
+			}
```

Comment:
Actually we can reduce this into one line.
```suggestion
			require.Equal(t, test.wantInviteIdChanged, test.originalInviteId != team.InviteId)
```

Reply:
Only 1/5, but I think this makes it harder to understand and I don't really see what we gain from changing it.

Reply:
Heh, the one-liner was a little bit more descriptive to me, but not a big deal.